test/index-test.js

```diff
@@ -0,0 +1,67 @@
+/* global describe, it */
+require('babel-register')();
+
+var assert = require('assert');
+var shallow = require('enzyme').shallow;
+
+var reactExpressMiddleware = require('../');
+var serverRenderMiddleware = require('../lib/index.js').default;
+var clientRenderMiddleware = require('../lib/browser.js').default;
+
+var Container = require('./components/Container.js');
+
+describe('React Express Middleware', function () {
+	it('should return a function', function () {
+		assert.equal(typeof reactExpressMiddleware(), 'function');
+	});
+
+	it('should render passed Component (server)', function (done) {
+		// 1. Generate middleware with 'shallow' as renderMethod
+		var renderMiddleware = serverRenderMiddleware({
+			template: 'foo.html',
+			renderMethod: function (Component) {
+				return shallow(Component).html();
+			}
+		});
+
+		var res = {
+			render: function (path, data) {
+				assert.equal(path, 'foo.html');
+				assert.equal(data.content, '<div><h1>Basic headline.</h1><p>Basic paragraph.</p></div>');
+			}
+		};
+		// 2. Run middleware (server)
+		renderMiddleware({}, res, function (err) {
+			assert.ok(!err);
```

Comment:
Move 'done' into here as well, it is the only way to be sure this is actually called.